modules/standard/FileSystem.chpl

```diff
@@ -734,10 +799,20 @@ proc sameFile(out error: syserr, file1: string, file2: string): bool {
   return ret != 0;
 }
 
-/* Returns true if both pathnames refer to the same file or directory
-   (utilizing operating system operations rather than string ones), returns
-   false otherwise.  May generate an error message.
-   file1, file2: string representations of paths to be compared.
+/* Determines if both pathnames refer to the same file or directory (utilizing
+   operating system operations rather than string ones) and returns the result
+   of that check
```

Comment:
You might mention that the main reasons this is not the same as string comparison is paths with /../../ or symbolic links.